.gitmodules

```diff
@@ -2,3 +2,8 @@
 	path = secp256k1-kmp/native/secp256k1
 	url = https://github.com/bitcoin-core/secp256k1
 	branch = master
+
+[submodule "rust-ed25519-bip32"]
+	path = rust-ed25519-bip32
+	url = https://github.com/input-output-hk/rust-ed25519-bip32.git
+	branch = rust-uniffi
```

Comment:
are we ok with this pointing at a branch, or should we merge that branch to master and update this ?

Reply:
We are going to change it to `main` branch after this [pr](https://github.com/input-output-hk/rust-ed25519-bip32/pull/1) merge.